src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -236,9 +236,19 @@ public bool CanBeginSelect()
 
         protected virtual string FormatSelectionText<T>(IEnumerable<T> elements)
         {
-            return elements.Any()
-                ? System.String.Join(" ", SelectionResults.Take(20).Select(x=>x.ToString()))
-                : "";
+            if (elements.Any())
+            {
+                string text = string.Join(", ", SelectionResults.Take(20).Select(x => x.ToString()));
+
+                if (elements.Count() > 20)
+                {
+                    text += "...";
```

Comment:
Wondering if these extra dots will not go well with whatever processing someone is doing with the resulting string :).

Reply:
Hmm. Right. @mjkkirschner / @QilongTang , do we consider this a breaking change? If so I'll hold off, this is not a critical fix, just a nice-to-have in Player. We could perhaps also fix it in Player, but in my opinion it makes sense to fix here.

Reply:
If this is going to be part of the selection node value passed to e.g. watch node then yes, if the `...` is only part of the selection display then this should not impact IMHO

Reply:
👍 This does not affect the value - that's always the full list of elements. This affects `Selection`'s `Text` property, which is used in the node's display in canvas, and also by Dynamo Player to send a display value to Player client.

Reply:
But `Selection.Text` _is_ public, and this changes its output.

Reply:
I think I mean the value passed to downstream, not the list.. It might be good to provide an example:
![image](https://github.com/user-attachments/assets/c6b9622c-c7a2-4f43-b40f-1448d3794235)

In this case, the `Family Types` node display value becomes AC_BXXXX... while the downstream node not affected and also the preview bubble remains the same right?

Reply:
Oh I realize that this change is not what I thought it would be. Does not look like Dynamo side is impacted by this though..

Reply:
Right, not that. Here's what it looks like
![image](https://github.com/user-attachments/assets/9c5decee-a0b6-40ae-889f-ff8f829f4dca)
I also realized changes are needed in DynamoRevit since this method is overridden there. See https://github.com/DynamoDS/DynamoRevit/pull/3080.